### PR TITLE
add joss paper

### PR DESCRIPTION
**Pull request by paulflang**

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+title: 'Gala: A Python package for galactic dynamics'
```
Comment (ChrisRackauckas) [inline paper/paper.md]:
```suggestion
title: 'SBMLToolkit.jl: A Julia package for generating ModelingToolkit models from SBML'
```

File: paper/paper.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+authors:
+  - name: Paul F. Lang
+    orcid: 0000-0002-6388-2405
+    corresponding: true
+    affiliation: "1, 2" # (Multiple affiliations must be quoted)
+  - name: Anand Jain
+    affiliation: 1
+  - name: Author with no affiliation
+    corresponding: true # (This is how to denote the corresponding author)
+    affiliation: "1, 3"
```
Comment (ChrisRackauckas) [inline paper/paper.md]:
```suggestion
  - name: Chris Rackauckas
    affiliation: "1, 3"
```

File: paper/paper.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+and accelerating numerical analysis and computational science. Building on the
+computer algebra system Symbolics.jl [@10.1145/3511528.3511535], many high-performing solvers for 
+differential equations have been developed in Julia [@DifferentialEquations.jl-2017]. In
+particular the Scientific Machine Learning (SciML) ecosystem of Julia packages
+includes ModelingToolkit.jl [@ma2021modelingtoolkit], a modeling framework for high-performance
+symbolic-numeric computation in scientific computing and scientific machine learning.
+It allows for users to give a high-level description of a model for symbolic
+preprocessing to analyze and enhance the model. ModelingToolkit can automatically
+generate fast functions for model components, along with automatically sparsifying
+and parallelizing the computations. This enabled highly performing solvers of
```
Comment (ChrisRackauckas) [inline paper/paper.md:49]: Maybe composition should be mentioned? Though we haven't actually used it with SBML

Comment (paulflang) [reply paper/paper.md:49]: Yeah, I don't think SBML is well suited for that. It is too often that the semantics of species in two different SBML models differs slightly (different isoform, complex composition, posttranslational modification), even if the models are for the same cell type to begin with.

**Comment by paulflang** [comment]
Hi @ChrisRackauckas and @anandijain . I am attaching the compiled PDF for the paper. Let me know if you are happy with it. @anandijain : do you have an orcid I should add?
[paper.pdf](https://github.com/SciML/SBMLToolkit.jl/files/11045844/paper.pdf)

**Comment by ChrisRackauckas** [comment]
I think this looks good to go. CI is failing on v1.6 though? 😅

**Comment by paulflang** [comment]
@ChrisRackauckas or @anandijain  : Do you have an idea where the ` LoadError: Cert verify failed: BADCERT_EXPIRED while requesting` in the CI is coming from?

**Comment by ChrisRackauckas** [comment]
I don't know what that is, maybe @thazhemadam might.

**Pull request merged**